src/dyad/core/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+set(DYAD_CORE_SRC ${CMAKE_CURRENT_SOURCE_DIR}/dyad_core.c)
+set(DYAD_CORE_PRIVATE_HEADER ${CMAKE_CURRENT_SOURCE_DIR}/dyad_envs.h)
+set(DYAD_CORE_PUBLIC_HEADER ${CMAKE_CURRENT_SOURCE_DIR}/dyad_core.h)
+
+add_library(dyad_core SHARED ${DYAD_CORE_SRC})
+target_link_libraries(dyad_core PUBLIC Jansson::Jansson flux::core)
+target_link_libraries(dyad_core PRIVATE utils murmur3 dyad_dtl)
```

Comment:
In the `dtl` directory, the CMake target is named `${PROJECT_NAME}_dtl`. We should use the same here

Reply:
Fixed